app/controllers/application_controller.rb

```diff
@@ -9,6 +9,8 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  # overrides 'route_translator' method to reset locale to English
+  skip_around_action :set_locale_from_url
```

Comment:
interesting that this is needed

Reply:
that method resets the locale to English if there is an error in any devise form